### ISSUE TITLE
Identification of infected person

### ISSUE THREAD

**Issue by gagamail** [corona-warn-app/cwa-wishlist#185]
Not sure where the best place for this topic is.

At work it was possible that we identified an infected person with the help of the app. The problem: Almost everyone has the app installed and of course you talk with others about it. In our case a lot of people got a warning at the same time. After talking to each other we realized that only people around one specific office got warnings (not all people had direct contact with the infected person, "contact" was detected through the walls)-> The person was identified.

Already thought about a solution for this problem but had no idea how to solve this problem without making the App not work anymore. But maybe someone else has an idea.

**Comment by daimpi** [comment]
This is unfortunately an inherent property of contact-tracing (no matter whether digital or manual). If a sufficiently large number of ppl share their contact status with each other de-anonymization becomes very likely even if the name of the infected person is never shared. The only way to prevent this would be to disallow ppl to share their contact status with others, which is non-enforceable. The other alternative would be to not do contact-tracing in the first place, but that isn't a reasonable option either given we're currently  in the middle of a pandemic.

**Comment by daimpi** [comment]
Related issue: https://github.com/corona-warn-app/cwa-documentation/issues/273

**Comment by schaupets** [comment]
Yes, with this kind of social interaction you can spot an infected person. But: 

- The infected person must go into quarantäne, so it will disappear from the office suddenly, which is a much clearer indication who is the infected.
- The infected person must give your name to the Gesundheitsamt as Kontakt. So all of you others in the office will be contacted, except the one infected.
- The Gesundheitsämter ask to inform the contact persons directly and personally, because they are overloaded. So if you are abled to identify the infected, the infected should also know you and inform you directly. 

So in my optinion  it makes no sense in closing a small door in the App while there is a big open door forced by legal requirements of  the „Infektionsschutzgesetz“. Even the “Ärztegeheimnis” is overruled by the Infektionsschutzgesetz.
